src/partials/header.html

```diff
@@ -0,0 +1,20 @@
+<div class="header-container">
+    <div class="header">
+        <nav class="header-nav">
+            <a href="#" class="header-nav__homePage-btn"><p>FILMOTEKA</p></a>
+                <svg class="header-nav__mobile-menu-icon" width="0" height="0" viewBox="0">
+                    <use href="#"></use>
+                </svg>
+            <ul class="header-nav__menu">
+                <li><a href="#"><p>HOME</p></a></li>
+                <li><a href="#"><p>MY LIBRARY</p></a></li>
+            </ul>
+        </nav>
+        <div class="header-input">
+            <input type="text" placeholder="Movie search" class="header-input__textBox">
```

Comment:
Z naszego konspektu wynika, że zawsze był stosowany form a w nim zagnieżdżony input. Trzeba też dodać button do wyszukiwania (ten SVG - lupa).
Zapytałem dodatkowo chatGPT i podpowiada:
W HTML tag input powinien być zazwyczaj używany wewnątrz tagu form. Tag form służy do grupowania elementów formularza, takich jak pola tekstowe, pola wyboru, przyciski, itp. i przesyłania ich wartości do serwera po wciśnięciu przycisku "submit". Tag input jest jednym z elementów formularza i służy do tworzenia różnych rodzajów pól formularza, takich jak pola tekstowe, pola wyboru, pola wyboru pliku, itp.
Umieszczanie tagu input poza tagiem form jest dozwolone, ale może to prowadzić do problemów z funkcjonalnością formularza, ponieważ wartości pól input nie zostaną przesłane do serwera wraz z innymi wartościami formularza, jeśli tag input nie jest umieszczony wewnątrz tagu form. Dlatego zaleca się umieszczanie tagu input wewnątrz tagu form dla większości przypadków użycia.

Reply:
zmienione